releasenotes/notes/fix-httpx-0.23.1-compatibility-99884a43d17b1222.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    httpx: Fixes an incompatibility from ``httpx==0.23.1`` when the ``URL.raw`` property is not available.
```

```suggestion
    httpx: fixes an incompatibility from ``httpx==0.23.1`` when the ``URL.raw`` property is not available.
```

I think the subject should always start with a lower case: https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#subject-case. The PR title should be updated to `chore(docs): add missing release note for #4595` to pass the pr naming ci check

---

https://github.com/DataDog/dd-trace-py/pull/4551/files#diff-e052d37005ef738791e533f325afeb1f194d5d2c18e5cb0cd992ca56097baba0R93

New release notes guidelines say to use sentence casing.